### PR TITLE
Containerise pipeline

### LINKED ISSUE #42: Dockerise the pipeline and push it to the ECR
Once the pipeline is finished, dockerise the pipeline so that it can run on the cloud.

### Steps
- [x] Create an ECR
- [x] Create a docker image of the pipeline with dockerfile
- [x] Push the docker image of 
- [x] Ensure it runs correctly

### User Story

As a developer, I want to ensure that my pipeline can run anywhere as a docker image, so that I can run and deploy it on the cloud without any problems.

### PR DESCRIPTION
**Pull request by e-lemma**
Closes #42 
I've created the Dockerfile for the pipeline and tested that it works. I also fixed the importing order of the pipeline script, and added a separate requirements.txt for the dockerfile to use. THANKS!

### CODE REVIEW

File: pipeline/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+python-dotenv
+pytest-cov
```
Comment (Faris-Abulula) [inline pipeline/requirements.txt:2]: Thanks for making this file, however should this be here?

**Review by Faris-Abulula — DISMISSED** [review]
One change and it's good

**Comment by e-lemma** [comment]
> One change and it's good

Good spot, removed it.

**Review by Zhi-704 — APPROVED** [review]
Looks good

**Review by Faris-Abulula — APPROVED** [review]
All good now thanks

**Pull request merged**